scripts/release_info.py

```diff
@@ -191,41 +194,44 @@ def main():
         current_release = _run_cmd(GIT_CMD_GET_CURRENT_VERSION)
         logs = _gather_logs(current_release)
         issue_links = _extract_issues(logs)
-        version_changes = _check_versions(current_release)
+        feconf_version_changes = _check_versions(current_release)
         setup_changes = _check_setup_scripts(current_release)
         storage_changes = _check_storage_models(current_release)
 
     summary_file = os.path.join(os.getcwd(), os.pardir, 'release_summary.md')
     with open(summary_file, 'w') as out:
         out.write('## Collected release information\n')
-        out.write('\n### Feconf version changes:\n')
-        for var, status in version_changes.items():
-            out.write('* %s: %s\n' % (var, status))
+
+        if feconf_version_changes:
+            out.write('\n### Feconf version changes:\nThis indicates that a '
+                      'migration may be needed\n\n')
+            for var in feconf_version_changes:
+                out.write('* %s  \n' % var)
 
         if setup_changes:
             out.write('\n### Changed setup scripts:\n')
             for var in setup_changes.keys():
-                out.write('* %s\n' % var)
+                out.write('* %s  \n' % var)
```

Comment:
not to versed in markdown,  i did notice that sometimes linebrakes aren't preserved if the trailing white space is missing 

although now that you mentioned it this might not be needed for a bullet list